docs-en/chapter_array_and_linkedlist/list.md

```diff
@@ -1,21 +1,21 @@
 # List
 
-A "list" is an abstract data structure concept, representing an ordered collection of elements. It supports operations like element access, modification, addition, deletion, and traversal, without requiring users to consider capacity limitations. Lists can be implemented based on linked lists or arrays.
+A "list" is an abstract data structure concept, representing an ordered collection of elements. It provides operations such as element access, modification, addition, deletion, and iteration, without requiring users to consider capacity limitations. Lists can be implemented using linked lists or arrays.
 
-- A linked list naturally functions as a list, supporting operations for adding, deleting, searching, and modifying elements, and can dynamically adjust its size.
-- Arrays also support these operations, but due to their fixed length, they can be considered as a list with a length limit.
+- A linked list inherently serves as a list, allowing for the addition, deletion, searching, and modification of elements. Additionally, it has the flexibility to adjust its length dynamically.
+- Arrays also provide these operations, but due to their immutable length property, they can be considered as a list with a length limit.
```

Comment:
honestly speaking this is good. However part of the requirements is to prioritize or simplicity so I think is original is better: "Arrays also support these operations, but due to their fixed length, they can be considered as a list with a length limit."